protocols/FarmLab/README_OT2.md

```diff
@@ -0,0 +1,40 @@
+# Indirect ELISA
```

Comment:
You can't have two separate readmes, hence this name should be 'README.md'. Otherwise it will break the protocol parser.